drivers/include/net/netdev.h

```diff
@@ -297,7 +297,11 @@ typedef struct netdev_driver {
      * @pre `(dev != NULL) && (iolist != NULL`
      *
      * @param[in] dev       Network device descriptor. Must not be NULL.
-     * @param[in] iolist    io vector list to send
+     * @param[in] iolist    IO vector list to send. Elements of this list may
+     *                      have iolist_t::iol_data == NULL or
+     *                      iolist_t::iol_size == 0. However, unless otherwise
```

Comment:
why is this again?

Reply:
Why is what?

Reply:
For the first half of the line see #11163, for the second see https://github.com/RIOT-OS/RIOT/pull/11187#pullrequestreview-214930207.